deploy/gcp/main.tf

```diff
@@ -295,6 +295,7 @@ until kubectl get po -n tidb -lapp.kubernetes.io/component=tidb | grep Running;
   echo "Wait for TiDB pod running"
   sleep 5
 done
+kubectl get pv -l app.kubernetes.io/namespace=tidb -o name | xargs -I {} kubectl patch {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
```

Comment:
This has to be done on teardown because new PVC will be created.

Reply:
The local storage will be deleted when the nodes are deleted. If you want to make persistent disk storage get deleted when pvc is deleted, you can just create the storage class with `Delete` reclaim policy.

Reply:
Besides, why changing the reclaim policy here? I've seen this already done in the above https://github.com/pingcap/tidb-operator/pull/493/files#diff-309b818571e8d7f3366262580cdb73e5R257

Reply:
One of those lines was left over, after talking on slack with @gregwebs it made sense to change the reclaim policy before `terraform destroy`

Reply:
I would like to solve this in the storageclass. But we haven't done that previously I believe to help prevent accidental data destruction. But I don't know if that is actually helpful since we already replicate data. This would only prevent accidental deletion in the case of TiKV losing 3+ pods.